rust_src/src/frames.rs

```diff
@@ -508,4 +508,22 @@ pub fn previous_frame(frame: LispFrameOrSelected, miniframe: LispObject) -> Lisp
     }
 }
 
+/// Mark FRAME as made.
+/// FRAME nil means use the selected frame.  Second argument MADE non-nil
+/// means functions on `window-configuration-change-hook' are called
+/// whenever the window configuration of FRAME changes.  MADE nil means
+/// these functions are not called.
+///
+/// This function is currently called by `make-frame' only and should be
+/// otherwise used with utter care to avoid that running functions on
+/// `window-configuration-change-hook' is impeded forever.
+#[lisp_fn]
+pub fn frame_after_make_frame(frame: LispFrameOrSelected, made: LispObject) -> LispObject {
```

Comment:
`made` could be a `bool` instead (with the function returning `bool`).

Reply:
But it is passed through to further code. So it can't be.

Reply:
It's used as as boolean and then returned, so `made: bool` and returning `bool` should work just fine.

Reply:
The problem is that non-nil is t in elisp and non-nil would be returned. So if `made` is "foo" it would return t instead of "foo".

Reply:
Fair point, it's returning the same object which could be anything. The documentation doesn't mention what it returns so it's all rather underspecified.